server/game/cards/12-SoW/IuchiRimei.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class IuchiRimei extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Move an attachment',
+            target: {
+                cardType: CardTypes.Attachment,
+                controller: Players.Opponent,
+                cardCondition: card => card.costLessThan(2),
+                gameAction: AbilityDsl.actions.selectCard(context => ({
+                    cardCondition: card => card !== context.target.parent && card.controller === context.target.parent.controller && card.type === CardTypes.Character,
+                    message: '{0} moves {1} to {2}',
+                    messageArgs: card => [context.player, context.target, card],
+                    gameAction: AbilityDsl.actions.ifAble(context => ({
```

Comment:
this should just be an `attach` action

Reply:
Removing the `ifAble` bit causes you to not be able to move attachments that would end up being illegal.  If I make this just `attach`, then the Force of the River test fails.